docs(reference/reply): When using async-await, need return (#4429)
fastify/fastify: docs/Reference/Reply.md

  reply.send(stream)
})
```

> When using async-await you will need to return the reply object:

Reviewer: @climba03003, both options returns Promise, the difference only is what inside - nothing or reply object, both of them handled same way and this part confuses me a bit - why I need to return reply if I can just await it?

```suggestion
When using async-await you will need to return or await the reply object:
```

Reviewer 2: The difference is just `implicit` or `explicit` handle of `Promise`.
`return` or `await` both `await`ing `reply` in the end.

The `await` here is used to signal the `payload` is not ready yet if you do something off the track.

  reply.send(stream)
})
```

> When using async-await, please return the reply object to signal fastify wait 
> for your further response.

Reviewer 3: ```suggestion
When using async-await you will need to return the reply object:
```

})
```

> When using async-await, please return the reply object to signal fastify wait 
> for your further response.

Reviewer 3: ```suggestion
When using async-await you will need to return the reply object:
```

```js
const fs = require('fs')

> fastify.get('/streams', function (request, reply) {

Reviewer 3: ```suggestion
fastify.get('/streams', async function (request, reply) {
```
